### PR TITLE
Account type added on registration controller

### PR DESCRIPTION
**Pull request by chinexman**
### What this PR does (required):
Account type added on registration controller to alter their account type if  the  user is a petSitter when creating a new profile 
### Any information needed to test this feature (required):
 during sign up as a new user, click on the become a sitter button  from the navbar menuitems

### Any issues with the current functionality (optional):
-

### CODE REVIEW

**Comment by wiroger9595** [comment]
you need to fix the conflicts before ethan check the review

File: server/controllers/auth.js [hunk 1]
```diff
@@ -32,7 +34,9 @@ exports.registerUser = asyncHandler(async (req, res, next) => {
   if (user) {
     await Profile.create({
       userId: user._id,
-      name
+      name,
+      accountType: accountType ? accountType : 'petOwner'
+
```
Comment (MUGABA) [inline server/controllers/auth.js:39]: This is good work, ternary operators, Though I still think there is more attributes, on the profile mode

**Review by MUGABA — CHANGES_REQUESTED** [review]
I think there is a profile controller where in my opinion you might need to update the profile from! Please check with @moffatethan  and see whether what am saying is okay, do not take my comment for it

**Comment by MUGABA** [comment]
And once again link your issue and fill the pr template

**Comment by moffatethan** [comment]
Need to close this PR - we're going to be taking a different approach which makes this work unnecessary. 👍🏻

**Pull request closed without merging**